.travis.yml

```diff
@@ -0,0 +1,12 @@
+
+services:
+- docker
+jobs:
+  include:
+    - stage: Build test container
+      script:
+        - docker build --target test --tag todo-app:test .
+        - docker run todo-app:test test_app.py
+        - docker run --env-file ./.env.test todo-app:test test_integration.py
+        - docker run -e SECRET_KEY -e API_KEY -e TOKEN -e ID_BOARD todo-app:test test_selenium.py 
```

Comment:
This is fine as we won't be adding anymore tests, but in a real project where you end up with many test files, you would want to group them into folders rather than having one command per file.